### ISSUE TITLE
How to build this source with cudnn v4, cuda 7.0

### ISSUE THREAD

**Issue by joyyang1215** [ShuangLI59/person_search#30]
I'd like to implement this source on TX1, as the newest version of cuda on TX1 is 7.0.
So that, the cudnn version is still v4.
Could yo tell me how to change the minimum version of cudnn while building.

**Comment by Cysu** [comment]
I'm afraid that it would not be an easy task. We upgraded from cudnn v4 to v5 in this [pull request](https://github.com/yjxiong/caffe/pull/71/files). You may need to manually revert those commits. Also please pay attention that the batch normalization layer is different between v4 and v5. Cudnn v4 uses inverse std, while v5 uses variance. We stored the variance in our pretrained models. You may need to convert it to cudnn v4 compatible version.

**Comment by joyyang1215** [comment]
thank you so much.
